nvdaHelper/remote/ia2LiveRegions.cpp

```diff
@@ -193,6 +193,34 @@ bool getTextFromIAccessible(wstring& textBuf, IAccessible2* pacc2, bool useNewTe
 	return gotText;
 }
 
+bool checkAccessibleStateAndRole(IAccessible* pacc, VARIANT varChild) {

```

Comment:
Either change the name to something a little more meaningful (I.e. something about live regions in the name) or at least a comment above stating this is to filter for live regions.

Reply:
Ah, isn't this entire file concerned only with live regions? Done.